corehq/apps/es/management/commands/reindex_icds_xforms.py

```diff
@@ -0,0 +1,56 @@
+from django.core.management.base import BaseCommand
+
+from corehq.elastic import get_es_new
+from corehq.util.dates import iso_string_to_datetime
+from elasticsearch.helpers import reindex
+
+
+class Command(BaseCommand):
+    help = ("Adhoc command for ICDS xforms reindex using scan and bulk insert API")
+
+    def add_arguments(self, parser):
+        parser.add_argument('index_name')
+        parser.add_argument('start_date', type=int)
+        parser.add_argument('end_date', type=int)
+
+    def handle(self, index_name, start_date, end_date, **options):
+        es = get_es_new()
+        start_date = self._get_last_start_date(es, start_date, end_date)
```

Comment:
why is this necessary?

Reply:
This allows the resume feature by skipping already inserted docs (for example when the command needs to be restarted after it exists due to node being down)